### DISCUSSION TITLE
SCP Labrat Eye Tracking Mod No Longer Compatible

### DISCUSSION THREAD

**Discussion by Furfuryl-Furfurate** [200Tigersbloxed/UnityMods#18] [General]
Thought I'd mention that SCP Labrat recently got a major overhaul, such that the eye tracking mod no longer works with it from what I can tell.

**Comment by 200Tigersbloxed** [comment]
I am aware of the 2.0 Update. I'm currently working on bigger projects, but I will try and overhaul my SCPL mods eventually
